CHANGES.md

```diff
@@ -29,6 +29,7 @@ Change Log
 * `Camera.flyTo` now honors max and min zoom settings in `ScreenSpaceCameraController`.
 * Fixed flying to `latitude, longitude, height` in the Geocoder.
 * Upgraded Knockout from version 3.2.0 to 3.4.0.
+* Added `DataSouceDisplay.ready`
```

Comment:
You should explain what `ready` actually means in this case.  i.e. ready is true whenever any static data associated with the Entity API has been rendered and false when data is waiting to be rendered.